### PR TITLE
Support for Android 13 and small corrections

### PR DESCRIPTION
**Pull request by nsknyc**
Tested on Android 7 DJI Smart Controller, and Samsung Tab S8 Ultra.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdk = 34
+    compileSdk 33
```
Comment (Peter-St) [inline app/build.gradle:12]: Is it required to drop down the SDK version to 33?
Are there any ways to keep SDK 34 and run it successfully on Android 13 devs?

**Comment by Peter-St** [comment]
I welcome your decision to commit to Android 13 permissions in the Repo.
I will merge this request to fix this permission issue.

**Comment by nsknyc** [comment]
I believe I had an older gradle with 28, and put 33. I can test 34 today no problem. This should technically allow android 14 users as well, but Samsung has not released an official upgrade in the US for the tab s8 for me to test it.

**Comment by nsknyc** [comment]
As expected SDK 34 no issues on a 13 device.

**Pull request merged**